src/Microsoft.ML.TimeSeries/IidSpikeDetector.cs

```diff
@@ -171,10 +171,39 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, Dat
     }
 
     /// <summary>
-    /// The <see cref="IEstimator{ITransformer}"/> for detecting a signal spike on an
-    /// <a href="https://en.wikipedia.org/wiki/Independent_and_identically_distributed_random_variables"> independent identically distributed (i.i.d.)</a> time series.
-    /// Detection is based on adaptive kernel density estimation.
+    /// The <see cref="IEstimator{TTransformer}"/> to detect a signal spike on an
+    /// <a href="https://en.wikipedia.org/wiki/Independent_and_identically_distributed_random_variables"> independent identically distributed (i.i.d.)</a>
+    /// time series based on adaptive kernel density estimation.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// To create this estimator, use [DetectIidSpike](xref:Microsoft.ML.TimeSeriesCatalog.DetectIidSpike(Microsoft.ML.TransformsCatalog,System.String,System.String,System.Int32,System.Int32,Microsoft.ML.Transforms.TimeSeries.AnomalySide)).
+    ///
+    /// ### Input and Output Columns
+    /// There is only input column and its type is <xref:System.Single>.
+    ///
+    /// | Output Column Name | Column Type | Description|
+    /// | -- | -- | -- |
+    /// | All input columns | Any | All input columns would pass by without being modified.
+    /// | `Prediction` | Known-sized vector of <xref:System.Single> | It contains alert level (non-zero value means a change point), score, and p-value.
+    ///
```

Comment:
>   /// [](start = 1, length = 6)

if your transform requires additional nugets, please make sure to include it like this:
[https://github.com/dotnet/machinelearning/blob/1fd8971bd1f4bf967b06e6f40b3f7042d8bb9f4d/src/Microsoft.ML.LightGbm/LightGbmBinaryTrainer.cs#L100](https://github.com/dotnet/machinelearning/blob/1fd8971bd1f4bf967b06e6f40b3f7042d8bb9f4d/src/Microsoft.ML.LightGbm/LightGbmBinaryTrainer.cs#L100)